sys/vfs/vfs.c

```diff
@@ -1074,4 +1074,8 @@ void auto_init_vfs(void)
     }
 }
 
+clist_node_t *vfs_get_mounts(void)
+{
+    return _vfs_mounts_list.next;
```

Comment:
Why the new semi-public function compared to the existing vfs_iterate_mounts?

Reply:
I completely missed that `vfs_iterate_mounts()` exists :sweat_smile: